### DISCUSSION TITLE
Please post on https://forum.typst.app/ instead.

### DISCUSSION THREAD

**Discussion by laurmaedje** [typst/typst#4960] [General]
Link: https://forum.typst.app/

We'll keep the discussion here open for now, but might close them in the future.

Update April 2026: Discussions here are now fully closed.

**Comment by ouuan** [comment]
Could you explain the context and rationale behind this decision?

**Reply by laurmaedje** [reply]
- The prime reason for the new Forum is to have a lasting, searchable place for questions and answers. GitHub Discussion never really grew into that role.
- The new Forum is Discourse-based and much more featureful than GitHub discussions.
- GitHub can be alienating for less technical users.
- Various people do not want to create a GitHub account. The new forum uses Typst Accounts instead.
- As for why we might close GitHub discussions: Having two forums spreads information across them and generates less total value for the users of both.

**Reply by studyingegret** [reply]
Hello @laurmaedje, I'm still evaluating Typst, but I cannot access the given link at all (page loads forever). I guess it's due to my geolocation. I don't know if it's just me having this issue, but the page can't load with all the browsers I have. Can you take a look at it?

**Reply by studyingegret** [reply]
Ah! Never mind, it finally successfully loaded!

**Comment by gkutiel** [comment]
There is a problem to sign in to https://forum.typst.app
> Sorry, there was an error authorizing your account. Please try again.

**Reply by studyingegret** [reply]
Hi, I'm able to sign up and log in, though I had to wait for a *significant* amount of time the first time the page loads. Maybe try again? The forum has a contact typst.app/contact
